src/js/personalization/dashboard-beta/reducers/index.js

```diff
@@ -1,7 +1,18 @@
 import claimsAppeals from '../../../claims-status/reducers';
-import prescriptions from '../../../rx/reducers';
+import prescriptions from '../../../rx/reducers/prescriptions';
+import recipients from '../../../messaging/reducers/recipients';
+import folders from '../../../messaging/reducers/folders';
+import { combineReducers } from 'redux';
```

Comment:
Nice! This is cool. Definitely illustrates that whatever we do with the platform initiative we will have to make sure we can continue to share code cross-apps

Reply:
Yep. DRY!